stdlib/task/src/main/ballerina/src/task/scheduler.bal

```diff
@@ -40,10 +40,10 @@ public type Scheduler object {
         }
     }
 
-    # Detach the provided `service` from the task.
+    # Detaches the provided `service` from the task.
     #
-    # + attachedService - ballerina `service` object which needs to be detached from the task.
-    # + return - Returns `task:SchedulerError` if the process failed due to any reason, nil otherwise.
+    # + attachedService - Ballerina `service` object which needs to be detached from the task
+    # + return - `task:SchedulerError` if the process failed due to any reason, nil otherwise
```

Comment:
```suggestion
    # + return - `task:SchedulerError` if the process failed due to any reason, nil otherwise
```
Same as above. Also, please refer the template for the correct format of the return statement description.

Reply:
Fixed in all places